components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -14,46 +14,71 @@ toc = true
 Hello and welcome to Chef Automate 2! This document guides you through the initial installation and trial activation.
 
 The easiest way to start exploring Chef Automate is to install it on a local VM.
-We recommend using [Vagrant][] to create your test environment.
+We recommend using [Vagrant](https://www.vagrantup.com/) to create your test environment.
 Use this Vagrantfile to provision a VM. Chef recommends a minimum of 4GB RAM.
 
+## Add an entry to /etc/hosts
+
+The example Vagrantfile sets the VM hostname to `chef-automate.test`. The installer uses this hostname as the FQDN that Chef Automate listens on.
+For security reasons, you must access Chef Automate's web UI using the configured FQDN.
+To avoid configuring DNS for your test instance, we recommend adding an entry to your workstation's `/etc/hosts` file.
+Use the following command to append the required entry:
+
+```bash
+echo 192.168.33.199 chef-automate.test | sudo tee -a /etc/hosts
+```
+
 ## Create a Vagrantfile
 
 Copy and paste the command below to create a `Vagrantfile`:
 
 ```ruby
-cat > Vagrantfile <<EOH
-# Put the contents of this snippet in a file named Vagrantfile
+cat > Vagrantfile <<'EOH'
+CFG_BOX      = 'bento/ubuntu-18.04'
+CFG_IP       = '192.168.33.199'
+CFG_HOSTNAME = 'chef-automate.test'
+
+$deployscript = <<-SCRIPT
+export DEBIAN_FRONTEND='noninteractive'
+apt-get -qq update
+apt-get install -y --no-install-recommends apt-transport-https ca-certificates curl gnupg-agent software-properties-common unzip
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+apt-get update
+apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io
+apt-get clean
+groupadd docker
+usermod -aG docker vagrant
```

Comment:
I'm not sure any of the above is needed for automate. It shouldn't rely on docker

Reply:
@avoidik would you mind updating your pr to remove these lines?

Reply:
@susanev done